content/en/docs/reference/kubernetes-api/common-definitions/list-meta.md

```diff
@@ -44,11 +44,5 @@ ListMeta describes metadata that synthetic resources must have, including lists
 
 - **selfLink** (string)
 
-  selfLink is a URL representing this object. Populated by the system. Read-only.
-  
-  DEPRECATED Kubernetes will stop propagating this field in 1.20 release and the field is planned to be removed in 1.21 release.
-
-
-
-
+  SelfLink is DEPRECATED read-only field that is no longer populated by the system.
```

Comment:
nit: I might call this an unused and / or legacy field; “deprecated” means “you should stop using this” (ahead of future removal, etc), whereas the truth now is that it already gone. The field is retained to meet API compatibility promises but the only field value is `""`.